vdev/src/commands/install_git_hooks.rs

```diff
@@ -0,0 +1,40 @@
+use anyhow::{Result, Ok};
+use std::process::Command;
+use crate::git;
+use crate::app::CommandExt as _;
+// use std::{env, path::PathBuf};
+
+/// Install a Git commit message hook that verifies
+/// that all commits have been signed off.
+#[derive(clap::Args, Debug)]
+#[command()]
+pub struct Cli {
+    /// The mode argument. Can be used to control which hook(s) are installed,
+    /// with the default being to install all available hooks.
+    mode: Option<String>
```

Comment:
If this only has a fixed set of values, you can use an `enum` instead and let clap do the validation. [See here](https://docs.rs/clap/latest/clap/_derive/_tutorial/index.html#enumerated-values)

Reply:
Since there is only one script at the moment I opted to remove the field